command/agent/config-test-fixtures/basic.hcl

```diff
@@ -40,6 +40,7 @@ client {
 	network_interface = "eth0"
 	network_speed = 100
 	cpu_total_compute = 4444
+	total_memory = 5555
```

Comment:
Instead of modifying the base test fixture, add a new hcl test case file that has this property set. This is so that we can exercise the default(total_memory not set) in the basic tests.

Reply:
This is only a parsing test. It checks that the hlc file is properly converted to the data structure. This is never going to run the detection of memory on the platform.

If the `memory_total_mb` is not set however, we should see a value set to `0` here instead.

Reply:
yes that's what I mean - let it assert 0 when total_memory is not set for basic.hcl.